--- /usr/include/boost/bimap/detail/debug/static_error.hpp   2008-03-22 17:45:55.000000000 -0400
+++ /usr/include/boost/bimap/detail/debug/static_error.rippled.hpp   2014-03-12 19:40:05.000000000 -0400
@@ -25,7 +25,6 @@
 // a static error.
 /*===========================================================================*/
 #define boost_bimap_static_error(message,variables)                           \
-        struct boost_pp_cat(bimap_static_error__,message) {};                 \
         boost_mpl_assert_msg(false,                                           \
                              boost_pp_cat(bimap_static_error__,message),      \
                              variables)
